Src/PerformanceCollector/Shared/Implementation/PerformanceCounterUtility.cs

```diff
@@ -33,7 +36,15 @@ internal static class PerformanceCounterUtility
             new Regex(
                 @"^\\(?<categoryName>[^(]+)(\((?<instanceName>[^)]+)\)){0,1}\\(?<counterName>[\s\S]+)$",
                 RegexOptions.Compiled);
-        
+
+        private static readonly Regex DisallowedCharsInReportAsRegex = new Regex(
```

Comment:
this is old logic that was copied from this module to labs and than removed from this module. There are no more disallowed symbols in metric names